docs/UserGuide.md

```diff
@@ -299,7 +324,24 @@ Examples:
 * `c-find John` returns `john` and `John Doe`.
 * `c-find alex david` returns `Alex Yeoh`, `David Li`.<br>
 
-#### 4.5 Deleting a person : `c-delete`
+#### 4.7 Locating persons based on matching tags: `c-tag-find`
+
+Finds all contacts that contain the KEYWORD(s) in their tags.
+
+Format: `c-tag-find KEYWORD [MORE_KEYWORDS]`
+
+* The search is case-insensitive. e.g `Friday` will match `friday`.
+* The order of the keywords does not matter. e.g. `friday monday` will match `monday friday`.
+* Only the tag names are searched.
+* Only full words will be matched e.g. `PartTime` will not match `PartTimes`.
+* Employees whoes tag(s) matching at least one keyword will be listed in the `Employee Contact DeatailL` pane
```

Comment:
I think whoes is a typo?

Reply:
Yes indeed, thank you for pointing out. I've changed it in the latest commit.